src/app/index.ts

```diff
@@ -369,6 +361,56 @@ module.exports = yo.extend({
     this._exitProcess();
   },
 
+  _projectBothScriptTypes: function (input, jsTemplates)
+  {
+    // Loop through jsTemplates, which is a subset of tsTemplates, and see if the project type exists
+    for (var i = 0; i < jsTemplates.length; i++)
+    {
+      var element = jsTemplates[i];
+      if (_.toLower(input) == _.toLower(element)) {
+        return true;
+      } 
+    }
+    return false;
+  },
+
+  _isValidInput: function (input, inputArray, isHostParam) 
+  {
+    if (!isHostParam && _.toLower(input) == 'excel-functions')
+    {
+      input = excelFunctions;
+    }
+
+    // validate host and project-type inputs
+    for (var i = 0; i < inputArray.length; i++)
+    {
+      var element = inputArray[i];
+      if (_.toLower(input) == _.toLower(element)) {
+        if (isHostParam){
+          this.options.host = element;
+        }
+        else {
+          this.options.projectType = element;
+        }
+        return true;
+      }
+    }
+    return false;
+  },
+
+_projectFolderExists: function ()

```

Comment:
The function is checked if the project folder is not empty (which is good). However, you should update the function name to match what it really does.